plugin.rb

```diff
@@ -2,7 +2,7 @@
 # about: Send an email containing a survey upon topic solved
 # version: 0.1
 # authors: HMS Solution Center Americas
-# url: https://github.com/HMSAB/discourse-survey-monkey.git
+# url: https://github.com/hms-networks/discourse-survey.git
```

Comment:
I think I forgot to remove the .git at the end of the url. I will do that when i push any changes you request @oliver-walker-hms

Reply:
I didn't catch anything else so after you push that i'll approve it